### PR TITLE
Add test for udisks2

### PR DESCRIPTION
**Pull request by michaelgrifalconi**
Simple tests for udisks2 using udisksctl checking status, block device information and loop-setup

- Related ticket: https://progress.opensuse.org/issues/49163
- Needles: no needles
- Verification run: 
  - 15.2  x86   https://openqa.suse.de/tests/4411161#
  - 15.1 s390  https://openqa.suse.de/tests/4411162#

- Local verification:
  - 12.2           http://10.161.229.249/tests/98
  - 12.3           http://10.161.229.249/tests/97
  - 12.4           http://10.161.229.249/tests/96
  - 12.5           http://10.161.229.249/tests/95
  - 15              http://10.161.229.249/tests/94
  - 15.1           http://10.161.229.249/tests/93
  - 15.2           http://10.161.229.249/tests/92

### CODE REVIEW

**Comment by grisu48** [comment]
Congratulations on the nice work Micheal!

As a side remark/comment: Have you considered in including Tumbleweed as well? We have a [Factory first](https://opensource.suse.com/suse-open-source-policy) policy, which encourages us to test against Tumbleweed first. It would be great if we could include this test in `lib/main_common.pm` as well, so that openSUSE can also benefit from it :-)

File: tests/console/udisks2.pm [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+
+sub run {
+    select_console 'root-console';
+
+    # Install udisks2 package. mkisofs and util-linux for support packages
+    if (is_sle('>=15')) {
+        zypper_call('in mkisofs udisks2 util-linux');
```
Comment (okurz) [inline tests/console/udisks2.pm:26]: Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

File: tests/console/udisks2.pm [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    # Gets device path from udisks loop-setup output
+    my $device_path;
+    for my $line (split /\n/, $udloop_output) {
+        if ($line =~ /Mapped file\s+(.*?)\s+as\s(.*?)\./) {
+            $device_path = $2;
+        }
+    }
+
+    my $loop_dev =
```
Comment (okurz) [inline tests/console/udisks2.pm:73]: You should not rely on any return value of `assert_script_run` and you also don't use it. Looks like this is a left-over from development work. Maybe you were trying `script_output` in before? :)

**Comment by michaelgrifalconi** [comment]
Thanks @grisu48 and @okurz for the feedback!

I adapted the test and scheduling and tested on VM:
- TW http://10.161.229.249/tests/109#
- SLE15.1 http://10.161.229.249/tests/110#
- SLE12.5 http://10.161.229.249/tests/111#

Please let me know if I should also schedule some additional runs on openqa.suse.de and openqa.opensuse.org and also if I should squash the commits after your review :)

File: tests/console/udisks2.pm (hunk 2 above)
Comment (michaelgrifalconi) [reply tests/console/udisks2.pm]: You're right, that is not supposed to be there, sorry!

**Comment by michaelgrifalconi** [comment]
Moved sle test runs on filesystem job as well to be consistent with openSUSE runs.

15.1 run https://openqa.suse.de/tests/4424544#
tw run https://openqa.opensuse.org/tests/1326468#

**Pull request merged**